src/main/java/frc/robot/RobotContainer.java

```diff
@@ -191,6 +195,14 @@ private void configureButtonBindings() {
         drivetrain, shooterSubsystem, turretSubsystem, ledSubsystem, controlBindings.translationX(),
         controlBindings.translationY(), STOCKPILE_MID_RED, STOCKPILE_MID_BLUE, STOCKPILE_INTERPOLATOR, 0.3)));
     
+    controlBindings.babyBomber().ifPresent(trigger -> trigger.whileTrue(
+      new BabyBirdCommand(
+        turretSubsystem, shooterSubsystem).deadlineWith(new LEDBlinkCommand(ledSubsystem, FLASH_COLOR, 0.1))
+      .andThen(new ShootTeleopCommand(
+        drivetrain, shooterSubsystem, turretSubsystem, ledSubsystem, controlBindings.translationX(),
+        controlBindings.translationY(), STOCKPILE_MID_RED, STOCKPILE_MID_BLUE, STOCKPILE_INTERPOLATOR, 0.3)
+        .deadlineWith(new LEDAlternateCommand(ledSubsystem, Color.kBlue, Color.kOrange, Seconds.of(0.1))))));
```

Comment:
To match your description of the driver holding the button down and it shooting multiple notes, you'll need to add `.repeatedly()` to the end. `whileTrue` does not reschedule the command when it ends.